code/backend/package.json

```diff
@@ -15,6 +15,10 @@
   },
   "devDependencies": {},
   "scripts": {
+    "testl": "DEVELOPMENT=true node template-generation/test/test-template-generation.js",
+    "install":"npm install",
+    "start": "node server/index.js",
+    "backend": "npm run install && npm run start",
```

Comment:
Is `npm install` and `npm backend` necessary? The deployment script can do `npm i && npm start`.